extensions/32-bit-address-space/README.md

```diff
@@ -0,0 +1,58 @@
+# General Design
+
+**Extension State: Under Development**
+**Requires: CP1.14**
+**CP Bit: CP1.16**
+
+# Overview
+
+This extension adds a new concept of _modes_ to the ISA. Modes are used to control things about the processor
+state which are binary-incompatible - that is, binary code written for a mode other than the mode the
+processor is actually in will almost certainly behave incorrectly.
+
+In particular, this extension focuses on the _Real 32-bit address mode_.
+
+# Added Instructions
+
+None!
+
+# Modes
+
+CR `1110`, equivalently `cr14` or "the MODE control register," holds a value indicating the current
+_mode_ of the processor.
```

Comment:
This should also state that it's only accessible in system privilege mode

Reply:
If privileges are supported, sure.